tds-dll-common/src/main/java/tds/dll/common/performance/caching/impl/NameAwareRedisCacheManager.java

```diff
@@ -0,0 +1,43 @@
+package tds.dll.common.performance.caching.impl;
+
+import com.google.common.collect.ImmutableList;
+import org.springframework.data.redis.cache.RedisCacheManager;
+import org.springframework.data.redis.core.RedisTemplate;
+import tds.dll.common.performance.caching.CacheType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This implementation of a RedisCacheManager configures internal caches based upon the cache name.
+ */
+public class NameAwareRedisCacheManager extends RedisCacheManager {
```

Comment:
This is pretty much a carbon-copy of the NameAwareRedisCacheManager in tds-common that is used by the microservices.